moveit_ros/planning/kinematics_plugin_loader/src/kinematics_parameters.yaml

```diff
@@ -0,0 +1,24 @@
+kinematics:
+  kinematics_solver: {
+    type: string_array,
+    default_value: ["kdl_kinematics_plugin/KDLKinematicsPlugin"],
```

Comment:
I don't think you want this to be an array type. Was it an array type before?

Reply:
It was not an array type before but multiple solvers could be listed using white spaces.
The original code parses the string and splits the string using white space - https://github.com/ros-planning/moveit2/blob/main/moveit_ros/planning/kinematics_plugin_loader/src/kinematics_plugin_loader.cpp#L327-L339

In examples, I have not seen more than one solver be listed